src/routes/activityReports/handlers.js

```diff
@@ -51,6 +51,8 @@ async function sendActivityReportCSV(reports, res) {
   // if we have some rows, we need to extract a list of goals and objectives and format the columns
   if (csvRows.length > 0) {
     const goalsAndObjectives = extractListOfGoalsAndObjectives(csvRows);
+    const user = await userById(userId);
+    const isAdmin = user.permissions.find((permission) => permission.scopeId === ADMIN);
```

Comment:
This is ok for now, but maybe in the future we can take advantage of `src/policies/activityReport.js` to create a function e.g. `canPreview `or something similar.